azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunnerManager.java

```diff
@@ -193,7 +197,7 @@ public FlowRunnerManager(Props props, ExecutorLoader executorLoader,
         new JobTypeManager(props.getString(
             AzkabanExecutorServer.JOBTYPE_PLUGIN_DIR,
             JobTypeManager.DEFAULT_JOBTYPEPLUGINDIR), globalProps,
-            parentClassLoader);
+            getClass().getClassLoader());
```

Comment:
by parent the code was referring to `AzkabanExecServer` class. There was no code initializing `FlowRunnerManager` with a different classloader. It was a simple new call. This means that that classloader for `AzkabanExecServer` and `FlowRunnerManager` will always be the same. Therefore sending it via argument seems redundant. This is also true further down the flow but I didn't want to refactor too much in this change since the PR is for a different purpose.